source/administration/command-line-tools.rst

```diff
@@ -752,8 +752,8 @@ platform user invite
       sudo ./platform user invite user@example.com myteam
       sudo ./platform user invite user@example.com myteam1 myteam2
 
-platform user migrate_auth
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+platform user migrate_auth (to ldap)
```

Comment:
Maybe we keep the command name the same but have a subsection for migrating to LDAP, and another for migrating to SAML

Reply:
I though this way was clearer, but if you prefer I can try to split on sections.